express-api/src/services/projects/projectsServices.ts

```diff
@@ -816,6 +816,8 @@ const getProjects = async (filter: ProjectFilter) => {
     });
   }
 
+  const totalCount = await query.getCount();
+
```

Comment:
had to use a separate query here after the agencyId filters and before the page filtering.

Reply:
See the comment on the lines below. I think we could remove this if that works for you.